.rubocop.yml

```diff
@@ -47,13 +46,11 @@ Lint/UselessAccessModifier:
     - class_methods
 
 Metrics/AbcSize:
-  Max: 34 # RuboCop default 17
```

Comment:
I removed these because they were manually removed in the todo. Regenerating with them removed allows calculating the current high water mark in the todo for later cleanup